README.md

```diff
@@ -236,6 +236,7 @@ Because the automatic instrumentation runs in a different classpath than the ins
 | [gRPC](https://github.com/grpc/grpc-java)                                                                                             | 1.5+                           |
 | [Hibernate](https://github.com/hibernate/hibernate-orm)                                                                               | 3.3+                           |
 | [HttpURLConnection](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/HttpURLConnection.html)                     | Java 7+                        |
+| [http4k](https://http4k.org)                                                                                                          | 3.270.0+                 |
```

Comment:
Hey @daviddenton!

Not sure how best to differentiate between instrumentation provided by this repo and instrumentation provided directly by the library.

I do think it's great to start tracking this.

Maybe:

```suggestion
| [http4k<sup>&dagger;</sup>](https://http4k.org)                                                                                                          | 3.270.0+                 |
```

and then at the bottom of the table:

```
<sup>&dagger;</sup> OpenTelemetry support built directly into the library
```

Reply:
maybe change link to https://www.http4k.org/guide/modules/opentelemetry/?

and instead of above suggestion:

```
<sup>&dagger;</sup> OpenTelemetry support provided by the library
```